creator-node/src/services/sync/primarySyncFromSecondary.js

```diff
@@ -64,12 +64,18 @@ module.exports = async function primarySyncFromSecondary({
     )
 
     // TODO should be able to pass this through from StateMachine / caller
-    let userReplicaSet = await getUserReplicaSet({
-      wallet,
-      selfEndpoint,
-      logger,
-      libs
-    })
+    let userReplicaSet = [
+      ...new Set(
+        await getUserReplicaSetEndpointsFromDiscovery({
+          libs,
+          logger,
+          wallet,
+          blockNumber: null,
+          ensurePrimary: false,
+          myCnodeEndpoint: selfEndpoint
```

Comment:
is the only change here to pass in `selfEndpoint`
also is that intentional? it executes a very different flow internally

Reply:
was this instance of the function just not renamed?

Reply:
it was renamed in prior PR but thats unrelated - the change here from the diff on the left is that `myCnodeEndpoint` changed from `null` to `selfEndpoint`
thats the only one i noticed at least

Reply:
^ yep. confirmed the change is irrelevant. myCnodeEndpoint is only used if ensurePrimary=true. I'll remove it for clarity